README.md

```diff
@@ -0,0 +1,17 @@
+# PPP-Wikidata-SPARQL
+
+Tool to generate SPARQL queries based on natural language input,
+powered by [Wikidata](https://www.wikidata.org/) and [Platypus](https://projetpp.github.io/). 
+
+## Development
+
+Running this project this requires a web server and the [Composer](https://getcomposer.org/) tool.
+
+Navigate to the diretory where your web server serves files from, then run the following commands:
```

Comment:
I find PHP's built-in server to be sufficient and more convenient for small projects like this. What about something like this for the last section:

To install the application, clone the repository and run `composer install` within its root directory. You can then serve the application using your local web server or using PHP's built-in server: run `php -S localhost:8080 -t www/` and open `http://localhost:8080` in your browser.

Reply:
That's much better indeed! I've made the edits as suggested.